db/test/fns/queue_test.js

```diff
@@ -89,7 +89,7 @@ suite(testing.suiteName(), function() {
       });
     });
 
-    suite('ensure_task_group/get_task_group', function() {
```

Comment:
It's probably best to actually duplicate this suite.  That way the deprecated `get_task_group` is verified to still be functional, as well as the new function.

Reply:
sure! thanks